### ISSUE TITLE
battery feedback

### ISSUE THREAD

**Issue by Paul-301** [actuallymentor/battery#283]
An option to hide the icon in the menu bar would be very useful.

**Comment by Yazi27** [comment]
any updates?

**Comment by mirza-rizvi** [comment]
This would be very helpful. As I see only a battery icon without any info for this app.

Making the top bar show two separate battery icons.

**Comment by deyshin** [comment]
Wouldn't this mean that the app is working only as CLI? 
Perhaps this should be a request to create stand-alone app, not the menu bar app.

**Comment by mirza-rizvi** [comment]
No, I meant only hiding the icon. But from the app, it will open up in a window for GUI settings, for people who are not comfortable with CLI. 

Very small target market I guess.
